Servers/UI/OJS.Servers.Ui/Controllers/AccountController.cs

```diff
@@ -44,20 +56,35 @@ public async Task<IActionResult> Login([FromBody]LoginRequestModel model)
                 return this.RedirectToAction("Index", "Home");
             }
 
-            var signInResult = await this.signInManager
-                .PasswordSignInAsync(model.UserName, model.Password, model.RememberMe, false);
+            ExternalUserInfoModel? externalUser;
```

Comment:
IMO, add a check if the environment is `Production`. When not in `Production`, do not call **Platform** and use the judge internal DB.